src/utils/eventUtils.ts

```diff
@@ -4,13 +4,17 @@ export function createCellEvent<E extends React.SyntheticEvent<HTMLDivElement>>(
   event: E
 ): CellEvent<E> {
   let defaultPrevented = false;
-  return {
+  const cellEvent = {
     ...event,
     preventGridDefault() {
       defaultPrevented = true;
     },
     isGridDefaultPrevented() {
-      return defaultPrevented;
+      return this.isDefaultPrevented() || defaultPrevented;
```

Comment:
micro-optimisation 🤷‍♂️ 
```suggestion
      return defaultPrevented || this.isDefaultPrevented();
```